src/targets/common.ini

```diff
@@ -44,8 +44,9 @@ build_flags =
 src_filter = ${common_env_data.src_filter} -<ESP8266*.*> -<STM32*.*>
 lib_deps =
 	makuna/NeoPixelBus @ ^2.6.7
-	ottowinter/ESPAsyncWebServer-esphome @ ^1.3.0
+	ottowinter/ESPAsyncWebServer-esphome @ ^2.1.0
```

Comment:
I'm starting to think we should remove the `^` char so it's locked at the version we've tested with, for all libraries!

Reply:
Would make the build more stable. And after each release we could pull in the current version for each lib and test it

Reply:
Do we still want to lock the version, or merge as is?

Reply:
I'd say merge as is to 2.1, and we can do a full review for all libraries for master.